splunk_add_on_ucc_framework/global_config.py

```diff
@@ -122,16 +122,25 @@ def inputs(self) -> List[Any]:
             return self._content["pages"]["inputs"]["services"]
         return []
 
+    @property
+    def configuration(self) -> List[Any]:
```

Comment:
hm, should this be `tabs` and the one `config_tabs` would be `resolved_tabs`?

Reply:
The name `tabs` was an ambiguous name for the method. The reason for naming it `configuration` is to maintain parity in `global_config.py` - since we have an `input` method (which returns input data) and a `pages` method (which returns page data), I thought to rename `tabs` to `configuration`.